client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -87,18 +86,8 @@ export const useRequestColumns = () => {
         description: 'description.forecast-quantity',
         align: ColumnAlign.Right,
         width: 200,
-        accessor: ({ rowData }) =>
-          QuantityUtils.suggestedQuantity(
-            rowData.itemStats.averageMonthlyConsumption,
-            rowData.itemStats.availableStockOnHand,
-            maxMonthsOfStock
-          ),
-        getSortValue: rowData =>
-          QuantityUtils.suggestedQuantity(
-            rowData.itemStats.averageMonthlyConsumption,
-            rowData.itemStats.availableStockOnHand,
-            maxMonthsOfStock
-          ),
+        accessor: ({ rowData }) => rowData.suggestedQuantity,
```

Comment:
now that the API is providing updated values, these lines can be removed totally. If the `key` is the same as the prop, which it is in this case, you don't need the `accessor` to be specified